sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastHashJoinExec.scala

```diff
@@ -90,6 +90,10 @@ case class BroadcastHashJoinExec(
     case _ => false
```

Comment:
ah i see where is the problem. Even the join produce one row for one input, we still need to copy the result if child needs it. Since it's a broadcast join, we only need to check the streaming side?

Reply:
oh, ya. you're right. I'll drop the build side. thanks.